staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -220,6 +227,11 @@ func LogAnnotation(ae *auditinternal.Event, key, value string) {
 	if ae == nil || ae.Level.Less(auditinternal.LevelMetadata) {
 		return
 	}
+	// In mutating webhooks we persist webhook patch in audit annotation with patchAuditAnnotationPrefix.
+	// Since the patch mutates request body, we skip logging if audit level is less than Request.
```

Comment:
I cannot follow. Why do we change generic `LogAnnotation` logic because some plugin (webhook) does something?